core/linker/default.h

```diff
@@ -83,6 +83,11 @@ SECTIONS
         *(.ramfunc.*)
         *(.data)
         *(.data.*)
+
+        /* The __uvisor_export_table must be placed at the end of flash to work
+         * correctly. */
+        KEEP(*(.uvisor_export_table))
```

Comment:
@Patater I believe this is relocated to the wrong section. I believe you want to put that into .text ?